src/VisualStudio/Core/Def/ProjectSystem/BrokeredService/WorkspaceProject.cs

```diff
@@ -138,6 +154,12 @@ public Task SetDisplayNameAsync(string displayName, CancellationToken cancellati
             return Task.CompletedTask;
         }
 
+        public Task SetProjectHasAllInformationAsync(bool hasAllInformation, CancellationToken cancellationToken)
```

Comment:
@tmeschter: going to use this as the name rather than LastDesignTimeBuildSucceeded; the name matches the underlying Workspace code better, but there's also other cases this would get used even if the design time build specifically didn't fail, so the name is a bit more inclusive of all uses.

Reply:
Sounds good.